ssl/statem/statem_clnt.c

```diff
@@ -3456,6 +3456,11 @@ MSG_PROCESS_RETURN tls_process_hello_req(SSL *s, PACKET *pkt)
         return MSG_PROCESS_ERROR;
     }
 
+    if ((SSL_get_options(s) & SSL_OP_NO_RENEGOTIATION)) {
```

Comment:
Here we use SSL_get_options() and elsewhere we use s->options. Which should we use?